content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml

```diff
@@ -23,7 +23,8 @@ Communication protocols:
   Programmable Serial ports: RS485
 Power:
   Input voltage: 12-24V DC
-  Output voltage: 24V
+  Rated voltage: 250V AC
+  Maximum switching voltage: 400V AC
```

Comment:
```suggestion
  Output relay rated voltage: 250V AC
  Output relay maximum switching voltage: 400V AC
```

Reply:
I've just realized that we should explain that those two specs are about the outputs of the relays so I've added that suggestion. Please change it to it and merge!